src/BlockchainWriter/DAO.ts

```diff
@@ -32,6 +32,18 @@ export class DAO {
       blockHeight: null,
     })
 
+  readonly purgeStaleTransactions = (currentBlockHeight: number, maxBlockHeightDelta: number) =>
+    this.blockchainWriterCollection.updateMany(
+      {
+        txId: { $ne: null },
+        blockHeight: null,
```

Comment:
I am guessing $ne is not empty, can we use $ne for blockHeight as well?

Reply:
not equal

Reply:
we are looking for entries where the txId is not null and the blockHeight is null.

Reply:
ah yeah ok.